tests/e2e/e2e.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from "@playwright/test";
+
+test("Create an account", async ({ page }) => {
+  await page.goto("http://localhost:3030/login");
+  await page.getByRole("link", { name: "Continue with Email" }).click();
+
+  await page
+    .getByPlaceholder("Email Address")
+    .type(`test_${Math.random()}@test.com`);
+  await page.getByRole("button", { name: "Send a magic link" }).click();
+
+  await expect(page.getByText("Welcome to Trigger.dev")).toBeVisible();
+  await page.getByLabel("Full name").type("John Doe");
+  await page.getByRole("button", { name: "Continue" }).click();
+
+  await page.getByLabel("Organization name").type("Test Org");
+  await page.getByLabel("Project name").type("Test Project");
+  await page.getByRole("button", { name: "Create" }).click();
+
+  await expect(page.locator("h1").filter({ hasText: /^Jobs$/ })).toBeVisible();
+});
+
+test("Verify jobs from the test nextjs project", async ({ page }) => {
+  await page.goto("http://localhost:3030");
```

Comment:
Same here

Reply:
I've added the base URL to the config file and updated the tests to use the relative paths.